Makefile

```
@@ -9,6 +9,13 @@ CLIPPY_OPTS := -D warnings -D clippy::clone_on_ref_ptr -D clippy::enum_glob_use
 test: ## Run all tests.
 	cargo test ${VERBOSE} --all -- --nocapture
 
+cov: ## Run code coverage.
+	# Tarpaulin only supports x86_64 processors running Linux.
+	# https://github.com/xd009642/tarpaulin/issues/161
+	export PATH="$(pwd)/devtools/cov:$PATH"
+	# https://github.com/xd009642/tarpaulin/issues/190#issuecomment-473564880
+	RUSTC=rustc-proptest-fix taskset -c 0 cargo tarpaulin --all -v --out Xml
```

A quick fix, remove the `export`, and use absolute path for the script.

```suggestion
	RUSTC="$$(pwd)/rustc-proptest-fix" taskset -c 0 cargo tarpaulin --all -v --out Xml
```

---

"$$(pwd)/devtools/cov/rustc-proptest-fix"